src/Controller/UI/InformationProductSearchController.php

```diff
@@ -40,6 +40,8 @@ public function searchForInformationProduct(Request $request, InformationProduct
         $searchOptions->setResearchGroupFilter($researchGroupFilter);
         $searchOptions->setProductTypeDescFilter($productTypeDescFilter);
         $searchOptions->setDigitalTypeDescFilter($digitalTypeDescFilter);
+        $searchOptions->onlyPublishedInformationProducts(true);
+        $searchOptions->setMaxPerPage(1000);
```

Comment:
Should we set this back to 10?

Reply:
until we have pagination, and since we don't have that many info products currently, I think it might make sense to just show them all.